CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ Tests can be ran as usual from Xcode(Product -> Test or cmd+U). Snapshot tests s
 
 > Backpack team only
 
-To issue a new release make sure you've set the project up as above, that you have push acess to the Backpack CocoaPod and that you're logged in to CocoaPods trunk. `bundle exec pod trunk me` should print your info and including the Backpack pod in the output. Make sure you've added a new entry in the `CHANGELOG.md` with the version you're publishing then run `bundle exec rake release`.
+To issue a new release make sure you've set the project up as above, that you have push acess to the Backpack CocoaPod and that you're logged in to CocoaPods trunk. `bundle exec pod trunk me` should print your info and including the Backpack pod in the output. Make sure you've added a new entry in the `UNRELEASED.md` and update the `CHANGELOG.md` file with the version you're publishing then run `bundle exec rake release`.
```

Comment:
This is not correct, when releasing you should move the entries from `UNRELEASED.md` to `CHANGELOG.md` and add them under the header with the new version and then run `bundle exec rake release`